Sources/Valet/Accessibility.swift

```diff
@@ -21,14 +21,14 @@ import Foundation
 public enum Accessibility: Int, CaseIterable, CustomStringConvertible, Equatable {
     /// Valet data can only be accessed while the device is unlocked. This attribute is recommended for data that only needs to be accessible while the application is in the foreground. Valet data with this attribute will migrate to a new device when using encrypted backups.
     case whenUnlocked = 1
-    /// Valet data can only be accessed once the device has been unlocked after a restart. This attribute is recommended for data that needs to be accessible by background applications. Valet data with this attribute will migrate to a new device when using encrypted backups.
+    /// Valet data is accessible except between device restart and first unlock. This attribute is recommended for data that needs to be accessible by background applications. Valet data with this attribute will migrate to a new device when using encrypted backups.
```

Comment:
I find that repurposing Apple's documentation for `kSecAttrAccessibleAfterFirstUnlock` reads slightly better here. What do you think?

```suggestion
    /// Valet data cannot be accessed after a restart until the device has been unlocked once. This attribute is recommended for data that needs to be accessible by background applications. Valet data with this attribute will migrate to a new device when using encrypted backups.
```

I like this in that we're defining "first unlock" like we were in the previous comment, in case someone doesn't understand what that means.

Reply:
I like defining the first unlock bit as well. But I also want to be super explicit as to when the data is accessible. Thoughts on the below @sethfri?
```suggestion
    /// Valet data cannot be accessed after a restart until the device has been unlocked once; data is then accessible until the device is next rebooted. This attribute is recommended for data that needs to be accessible by background applications. Valet data with this attribute will migrate to a new device when using encrypted backups.
```

Reply:
That looks great @dfed!

Reply:
Removing the "then", and committing everywhere.
```suggestion
    /// Valet data cannot be accessed after a restart until the device has been unlocked once; data is accessible until the device is next rebooted. This attribute is recommended for data that needs to be accessible by background applications. Valet data with this attribute will migrate to a new device when using encrypted backups.
```